.circleci/config.yml

```diff
@@ -28,7 +28,7 @@ commands:
       ssh_user:
         type: string
     steps:
-    - run: '[[ $CIRCLE_BRANCH != "master" && -z $CIRCLE_PULL_REQUEST ]] && circleci step halt || true'
+    - run: '[[ ${CIRCLE_BRANCH} != "master" && -n ${CIRCLE_PULL_REQUEST} ]] && circleci step halt || true'
```

Comment:
Could you please replace the whole run step with:
```
          - run:
              name: Branch and PR filter
              command: |
                if [[ $CIRCLE_BRANCH != "master" && -z $CIRCLE_PULL_REQUEST ]] && circleci step halt || true
```

Reply:
actually you can also just leave it as it was previously:
`- run: '[[ $CIRCLE_BRANCH != "master" && -z $CIRCLE_PULL_REQUEST ]] && circleci step halt || true'`
Looks like this[ worked fine](https://circleci.com/workflow-run/af0c7f7c-b4f1-4071-bd8b-f204b49ebaaa) 2 weeks ago.